subcommittees/mentoring/membership-roles.md

```diff
@@ -0,0 +1,68 @@
+
+### Roles for any SWC/DC instructor or instructor trainee
+
+* [Discussion Session Participant](./)
+	- Join a discussion about teaching workshops.  
+* [Discussion Session Host/Co-Host](roles/discussion-hosts.md)
+	- Lead a discussion among fellow instructors about recent or upcoming workshops. 
+* [Mentor](roles/mentors.md)
+	- Work with a small group of new instructors to provide support and guidance.
+* [Mentee](roles/mentees.md)
+	- Work with a mentor and fellow new instructors to prepare for teaching the 
+	Software and Data Carpentry lessons, organizing workshops, and participation in the 
+	Software and Data Carpentry communities.  
+
+Anyone who is a certified instructor, or in the process of becoming certified 
+can participate in our activities in the roles above. 
+
+### Roles for mentoring committee members
+
+* [Chair and Vice-Chair](roles/chair-and-vice.md)
+	- Coordinate mentoring efforts, organize and lead mentoring committee meetings
+* [Secretary](roles/secretary.md)
+	- Take notes in mentoring committee meetings, submit official minutes
+* [Steering Committee Liaison](roles/liaison.md)
+	- Facilitate communication between the steering committee and mentoring committee
+* [Discussion Coordinators](roles/discussion-coordinators.md)
+	- Coordinate volunteer hosts for instructor discussion sessions, contact instructors
+
+These roles are filled by [members](#mentoring-committee-membership) of the 
+mentoring subcommittee, in order to support 
+the operation and activities of the subcommittee.  Current 
+members in these roles are listed in the [README](README.md)
+
+### Choosing Mentoring Committee Member Roles
+
+These roles are filled yearly, in February.  Members can volunteer for any role.  
+Roles are filled in the order listed above, first by consensus (if it exists) and 
+then by a majority vote over a weeklong period via the etherpad if there is not 
+consensus.  Whenever possible roles should
+ be filled by unique people; however, if there are not enough volunteers, 
+ one person may hold up to two positions
+
+In the case that a member is not able to fulfill the duties of their role for the 
+full term, a member will be chosen to fill that role until the time of the next 
+turnover.  
+
+## Mentoring Committee Membership
+
+To join the mentoring committee as a member, you must 
+
+- be either a: 
+	- certified instructor
+	- in the process of becoming certified (have attended instructor training or be confirmed to attend an upcoming training)
+- be active as an instructor/helper, either: 
+	- helped/instructed at a Software or Data Carpentry workshop in the past 8 months
+	- have helped or taught at over 5 Software/Data Carpentry workshops total
+- participate in mentoring committee activity, either: 
+	- attended at least 8 mentoring committee meetings in the past 12 months
+	- participated in mentoring committee activities (e.g. discussion sessions, mentoring program) in the past 
+4 months.  
+
```

Comment:
We haven't really voted - I became the chair because I was the only person who volunteered.  

I also want there to be as few hoops as possible, I guess I'm just trying to avoid two scenarios: 
- someone shows up to their first meeting and volunteers and then vanishes
- someone wants to vote and can't because they're not at the meeting

But we could address the first by just saying that if no one else (more qualified) volunteered, then too bad, and for the second, planning to vote via email/etherpad; you just need to be on the mentoring list so that you know about it.